docs/src/pages/[platform]/connected-components/storage/storagemanager/props.ts

```diff
@@ -20,9 +20,15 @@ const eventHandler = `(file: {key: string}) => void;`;
 
 export const STORAGE_MANAGER = [
   {
-    name: `accessLevel`,
+    name: `path`,
     description:
-      'Access level for files in Storage. See https://docs.amplify.aws/lib/storage/configureaccess/q/platform/js/',
+      'Path in s3 to put the file under. If `accessLevel` is undefined, then `path` must be provided, and can be either a string or callback. If `accessLevel` is provided, then `path` is optional and must be a `string`.',
+    type: `string | (input: { identityId: string | undefined }) => string`,
+  },
+  {
+    name: `accessLevel?`,
```

Comment:
Access level is still a required prop on the `StorageManagerProps` interface, it's what allows for a discrimnated union between the `accessLevel` and `path` use cases
```suggestion
    name: `accessLevel`,